<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hey! Healer 로그인</string>
  </property>
  <widget class="QWidget" name="q_heyhealer_login_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>300</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QLineEdit" name="q_heyhealer_id">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>아이디</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_heyhealer_pw">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>비밀번호</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_heyhealer_login_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="q_auto_save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>아이디 저장하기</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="q_heyhealer_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>로그인 해주세요</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>q_heyhealer_id</tabstop>
  <tabstop>q_heyhealer_pw</tabstop>
  <tabstop>q_auto_save</tabstop>
  <tabstop>q_heyhealer_login_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
